Version Bump (#82)
PennyLaneAI/pennylane-sf: .github/CHANGELOG.md

> # Release "0.19.0"

Reviewer: ```suggestion
# Release 0.19.0
```

Reviewer: Seems to be related to this line, correct?
https://github.com/PennyLaneAI/pennylane-sf/blob/master/.github/workflows/vb_script.py#L16

Reviewer 2: Ahh great catch! Not quite! It is a little more complicated since the `_version.py` file requires the quotes. I will make a patch for it once I finish with the demos

### Improvements

> * Change the name of `qml.Interferometer` to `qml.InterferometerUnitary` due to PennyLane deprecation

Reviewer: ```suggestion
* Changed the name of `qml.Interferometer` to `qml.InterferometerUnitary` due to PennyLane deprecation
```

> * Changed the name of `qml.Interferometer` to `qml.InterferometerUnitary` due to PennyLane deprecation

Reviewer: ```suggestion
* Changed the name of `qml.Interferometer` to `qml.InterferometerUnitary` due to a breaking change in PennyLane.
```
